src/QuestionInput.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+export default class QuestionInput extends Component {
```

Comment:
A lot of this component looks the same as your `AnswerInput`. Maybe you can define a generic `Input` component that takes the differences as props.